src/app/src/components/FacilityDetailSidebar.jsx

```diff
@@ -49,46 +68,119 @@ class FacilityDetailSidebar extends Component {
-                    </ul>
+                    <div className="control-panel__content">
+                        <ul>
+                            {
+                                error
+                                    .map(err => (
+                                        <li
+                                            key={err}
+                                            style={{ color: 'red' }}
+                                        >
+                                            {err}
+                                        </li>))
+                            }
+                        </ul>
+                    </div>
                 </div>
             );
         }
 
         if (!data) {
             return (
                 <div className="control-panel">
-                    {`No facility found for OAR ID ${oarID}`}
+                    <div className="control-panel__content">
+                        {`No facility found for OAR ID ${oarID}`}
+                    </div>
                 </div>
             );
         }
 
+        const facilityLat = last(data.geometry.coordinates);
+        const facilityLng = head(data.geometry.coordinates);
+
         return (
             <div className="control-panel">
-                {JSON.stringify(data)}
+                <div className="panel-header display-flex">
+                    <IconButton
+                        aria-label="ArrowBack"
+                        className="color-white"
+                        style={detailsSidebarStyles.backButtonStyle}
+                        onClick={goBack}
+                        disabled={fetching}
+                    >
+                        <ArrowBackIcon />
+                    </IconButton>
+                    <div>
+                        <h3 className="panel-header__title notranslate">
+                            {data.properties.name}
+                        </h3>
+                        <p className="panel-header__subheading notranslate">
+                            {data.properties.address}
+                        </p>
+                    </div>
+                </div>
+                <StaticMap
+                    lat={facilityLat}
+                    lng={facilityLng}
+                />
+                <div className="control-panel__content">
+                    <div className="control-panel__group">
+                        <h1 className="control-panel__heading">
+                            OAR ID: &nbsp;
+                        </h1>
+                        <p key="_id" className="control-panel__body">
+                            {data.properties.oar_id}
+                        </p>
+                    </div>
+                    <div className="control-panel__group">
+                        <h1 className="control-panel__heading">
+                            GPS Coordinates:
+                        </h1>
+                        <p key="geo" className="control-panel__body">
```

Comment:
These `key` props aren't hurting anything by remaining, probably, but they could also potentially be removed.

Reply:
Dropped in 49f35811acf9c8068c91fad80787c5ab9092ea53